ogr_fdw.c

```diff
@@ -1038,6 +1054,9 @@ ogrFeatureToSlot(OGRFeatureH feat, TupleTableSlot *slot, TupleDesc tupdesc)
 						break;
 					}
 					case OFTInteger:
+					#if GDAL_VERSION_MAJOR >= 2
+                        case OFTInteger64:
```

Comment:
This is #ifdef'ed against the GDAL version in ogr_fdw_info.c, it should be here too